README.md

```diff
@@ -69,22 +73,22 @@ If the project is now up and running for you locally, you might want to make a s
 
 The system:
 
-    You should fork the project from the original repo.
-    Get it running locally.
-    For new work create a new branch with an appropriate name.
-    Make changes, and commit.
-    Push the changes to your repo.
-    Make a PR from your repo to the original.
-    Respond to correspondence about the PR (if needed).
-    Make changes (if needed).
-    And on you go.. 
+    1. You should fork the project from the original repo.
+    2. Get it running locally.
+    3. For new work create a new branch with an appropriate name.
+    4. Make changes, and commit.
+    5. Push the changes to your repo.
+    6. Make a PR from your repo to the original.
+    7. Respond to correspondence about the PR (if needed).
+    8. Make changes (if needed).
+    9. And on you go.. 
```

Comment:
Please remove the indentation in each bullet point.

```suggestion
1. You should fork the project from the original repo.
2. Get it running locally.
3. For new work create a new branch with an appropriate name.
4. Make changes, and commit.
5. Push the changes to your repo.
6. Make a PR from your repo to the original.
7. Respond to correspondence about the PR (if needed).
8. Make changes (if needed).
9. And on you go.. 
```

Reply:
Done.